CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ Release date: TBD
 
 ### Improvements
```

Comment:
We could add this note: "Added `CTRL+SHIFT+MINUS` as a shortcut for zooming out"